<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chord Prog Syncopator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Petaluma Script</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Chord Progression Player/Syncopator</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="verticalFrame_2">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777206</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_test">
           <property name="text">
            <string>Test MIDI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>14</height>
            </size>
           </property>
           <property name="text">
            <string>Choose Midi Device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_device"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_refresh">
           <property name="text">
            <string>Refresh MIDI Devices</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>14</height>
            </size>
           </property>
           <property name="text">
            <string>Chose Bar Lenght</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_bar_length"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>14</height>
            </size>
           </property>
           <property name="text">
            <string>Chose Tempo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinbox_tempo"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_play">
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_stop">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_loop">
           <property name="text">
            <string>Toggle Looping</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>250</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>boogie.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Info:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="text_area">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>400</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_tonality">
          <property name="font">
           <font>
            <family>Noto Sans Black</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Chord Progression:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>1500</width>
            <height>0</height>
           </size>
          </property>
          <property name="scene" stdset="0">
           <string notr="true">scene</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
